codalab/model/worker_model.py

```diff
@@ -172,34 +181,14 @@ def get_workers(self):
                 'run_uuids': [],
                 'dependencies': row.dependencies
                 and self._deserialize_dependencies(row.dependencies),
+                'shared_file_system': row.shared_file_system,
             }
             for row in worker_rows
         }
         for row in worker_run_rows:
             worker_dict[(row.user_id, row.worker_id)]['run_uuids'].append(row.run_uuid)
         return list(worker_dict.values())
 
-    def get_bundle_worker(self, uuid):
```

Comment:
This was moved to `bundle_model` since this is more a property of the bundle (conceptually) and practically it was needed in there, and any client of this method has access to both bundle and worker models.